lib/core/Canvas.js

```diff
@@ -229,8 +228,14 @@ Canvas.prototype._init = function(config) {
     height: '100%'
   });
 
+  assignStyle(svg, {
+    outline: 'none'
+  });
+
   domAttr(svg, 'tabindex', 0);
 
+  const focus = this.focus.bind(this);
```

Comment:
Why do we do this? Usually we don't do `bind` shinanigans, but ask users to properly execute the actual method with the receiver, i.e. `canvas.focus()`.

Reply:
Leftover. Removed.